apps/voting/contracts/Voting.sol

```diff
@@ -311,14 +335,17 @@ contract Voting is IForwarder, AragonApp {
         }
     }
 
+    /**
+     * @dev Internal function to execute a vote. It assumes the queried vote exists.
```

Comment:
Really good to add this here. It's unrelated to this PR (and I have said this before) but I don't like how this is done, as it's prone to error. This is supposed to be a "safe" function but it's not as it could be writing to an uninitialized vote in the mapping. It could be ourselves if we ever change this contract or anybody else inheriting it as it's an `internal` function.

Reply:
I see your point, but ofc it's a trade off between that and optimizing gas costs, specially in those situations where we want to ensure multiple conditions. That's why I added this in the meantime, at least until we come up with a decision, I think it helps pointing that out.